boards/yunjia-nrf51822/include/periph_conf.h

```diff
@@ -21,26 +21,12 @@
 
 #include "periph_cpu.h"
 #include "periph_conf_common.h"
+#include "cfg_clock_16_1.h"
```

Comment:
The original config did not enable the LFCLK. I am actually not sure if there is a lf crystal on that board. By looking at it, I can't find one. So I'd say keep it save and better disable it?!